app/controllers/organisations_controller.rb

```diff
@@ -8,6 +8,6 @@ def show
     @organisation = Organisation.find_by_whitehall_slug!(params[:id])
     sites = @organisation.sites.includes(:hosts)
     extra_sites = @organisation.extra_sites.includes(:hosts)
-    @sites = sites.concat(extra_sites).sort_by!(&:abbr)
+    @sites = sites + extra_sites.sort(&:abbr)
```

Comment:
Is this sorting the sites as expected? Before we were concatenating the 2 lists before sorting them, but I'm not sure if that's what's happening now. It looks like it's sorting `extra_sites` and then adding sites to it, is that correct?